libs/nx/src/generators/ui/libs/ui-dialog-helm/files/lib/hlm-dialog-content.directive.ts.template

```diff
@@ -9,7 +9,7 @@ import { ClassValue } from 'clsx';
 export class HlmDialogContentDirective {
   private _inputs: ClassValue = '';
   private _statusProvider = injectExposesStateProvider({ host: true });
-  public state = this._statusProvider?.state ?? signal('closed').asReadonly();
```

Comment:
I remember there being a reason these were optional for some of them... I think it is because technically you can use the hlm components and directives without their counterparts. Let's say you just want to attach the same initial styles to something, but do not care about the status changes. I remember something along those lines

Reply:
The change here is reflected by the inject call right above it. If it is indeed `optional` then it should have `{ optional: true }`